hail/python/hailtop/hailctl/dev/benchmark/run/matrix_table_benchmarks.py

```diff
@@ -175,4 +181,19 @@ def per_row_stats_star_star():
 
 @benchmark
 def read_decode_gnomad_coverage():
-    hl.read_matrix_table(resource('gnomad_dp_simulation.mt'))._force_count_rows()
\ No newline at end of file
+    hl.read_matrix_table(resource('gnomad_dp_simulation.mt'))._force_count_rows()
+
+@benchmark
+def export_range_matrix_table_entry_field_p100():
```

Comment:
how long do these take? Might be a better benchmark to export rows/cols/GT from profile.mt

Reply:
What speed is required? Takes about 3 seconds for each iteration. IIRC, you said ten iterations? so 30 seconds for each benchmark.

```
In [3]: %%timeit 
   ...: mt = hl.utils.range_matrix_table(n_rows=1_000_000, n_cols=10, n_partitions=100) 
   ...: mt = mt.annotate_entries(x = mt.col_idx + mt.row_idx) 
   ...: mt.x.export('/tmp/foo')                                                                                                                                                                
[Stage 1:======================================================> (99 + 2) / 101]2019-08-23 11:02:47 Hail: INFO: merging 101 files totalling 72.3M...
2019-08-23 11:02:48 Hail: INFO: while writing:
    /tmp/foo
  merge time: 395.290ms
[Stage 3:======================================================> (98 + 3) / 101]2019-08-23 11:02:51 Hail: INFO: merging 101 files totalling 72.3M...
2019-08-23 11:02:51 Hail: INFO: while writing:
    /tmp/foo
  merge time: 153.281ms
[Stage 5:=====================================================>  (96 + 5) / 101]2019-08-23 11:02:54 Hail: INFO: merging 101 files totalling 72.3M...
2019-08-23 11:02:54 Hail: INFO: while writing:
    /tmp/foo
  merge time: 196.225ms
[Stage 7:===================================================>    (92 + 8) / 101]2019-08-23 11:02:57 Hail: INFO: merging 101 files totalling 72.3M...
2019-08-23 11:02:58 Hail: INFO: while writing:
    /tmp/foo
  merge time: 188.630ms
[Stage 9:===================================================>    (92 + 8) / 101]2019-08-23 11:03:01 Hail: INFO: merging 101 files totalling 72.3M...
2019-08-23 11:03:01 Hail: INFO: while writing:
    /tmp/foo
  merge time: 194.541ms
[Stage 11:=====================================================> (98 + 3) / 101]2019-08-23 11:03:04 Hail: INFO: merging 101 files totalling 72.3M...
2019-08-23 11:03:04 Hail: INFO: while writing:
    /tmp/foo
  merge time: 299.583ms
[Stage 13:=====================================================> (98 + 3) / 101]2019-08-23 11:03:06 Hail: INFO: merging 101 files totalling 72.3M...
2019-08-23 11:03:07 Hail: INFO: while writing:
    /tmp/foo
  merge time: 139.772ms
[Stage 15:================================================>      (89 + 8) / 101]2019-08-23 11:03:09 Hail: INFO: merging 101 files totalling 72.3M...
2019-08-23 11:03:09 Hail: INFO: while writing:
    /tmp/foo
  merge time: 157.606ms
3.09 s ± 304 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
```

Reply:
this seems fine!

Reply:
we can also tweak once we've got it in.